test/unit/visitor/defuse_analyze.cpp

```diff
@@ -7,7 +7,7 @@
 
 #include <catch/catch.hpp>
 
-#include "ast/program.hpp"
+#include "ast/all.hpp"
```

Comment:
nitpik : if we have "few" ast classes to use then we include individual headers and avoid `all.hpp` (which is very large).